Common/cpp/Tools/RuntimeDecorator.cpp

```diff
@@ -293,11 +293,11 @@ void RuntimeDecorator::decorateUIRuntime(
                   const jsi::Value &thisValue,
                   const jsi::Value *args,
                   size_t count) -> jsi::Value {
-    progressLayoutAnimationFunction(args[0].asNumber(), args[1].asObject(rt));
+    progressLayoutAnimationFunction(args[0].asNumber(), args[1].asObject(rt), args[2].asBool());
```

Comment:
If I remember correctly asBool was only added in RN 69 and we should use getBool instead